Zend/zend_inheritance.c

```diff
@@ -212,6 +212,18 @@ char *zend_visibility_string(uint32_t fn_flags) /* {{{ */
 }
 /* }}} */
 
+char *zend_asymmetric_visibility_string(uint32_t fn_flags) /* {{{ */
```

Comment:
You're returning an unmodifiable string

```suggestion
const char *zend_asymmetric_visibility_string(uint32_t fn_flags) /* {{{ */
```

Reply:
Looks like the function could be static as well